.travis.yml

```diff
@@ -56,6 +56,22 @@ jobs:
         on:
           repo: Yoast/wordpress-seo
           all_branches: true
+        provider: releases
+
+        on:
+          repo: Yoast/wordpress-seo
+          all_branches: true
+    # If the commit was tagged, create an artifact and push it to the distribution github
+      deploy:
+        script: grunt artifact
+        skip_cleanup: true
+        provider: script
+        script: bash scripts/deploy_to_dist.sh
```

Comment:
Can the Travis configuration have two `script` directives? Does Yaml even allow that?

Reply:
The travis linter does not complain and while testing we will see if it works